helpers.go

```diff
@@ -1,21 +1,25 @@
 package main
 
 import (
-	"errors"
+	"fmt"
 
 	"github.com/Cox-Automotive/alks-go"
 )
 
-func validateIAMEnabled(client *alks.Client) error {
+func validateIAMEnabled(client *alks.Client) *alks.AlksError {
 	// Validate STS for IAM active.
 	resp, err := client.GetMyLoginRole()
 	if err != nil {
 		return err
```

Comment:
```suggestion
		return err.Err
```

Reply:
client.GetMyLoginRole() returns an ALKSError so I thought to just pass that ALKSError through to where validateIAMEnabled gets called and return err.Err there